wiki/Beatmapping/Hitsound/vi.md

```diff
@@ -0,0 +1,48 @@
+---
+stub: true
+tags:
+  - hit normal
+  - hitnormal
+  - hit sound addition
+  - hit sound additions
+  - hitsound addition
+  - hitsound additions
+  - hit sounds
+  - hitsounds
```

Comment:
these tags need to be translated (while also keeping the originals). they help when searching for the article

Reply:
yes sure, I will translate them